openstack/cinder/templates/vcenter_datacenter_cinder_configmap.yaml

```diff
@@ -20,7 +20,7 @@ template: |
   data:
     cinder-volume.conf: |
       [DEFAULT]
-      enabled_backends = {{ .Values.backends.enabled }}
+      enabled_backends = {{ .Values.backends.enabled }}{{ if .Values.backends.vmware_fcd.enabled | default false }},vmware_fcd {{- end }}
```

Comment:
Why are you doing this so differently than the other backends? Could just add it to `backends.enabled` in `qa` and be done, right?

Reply:
done